Makefile

```diff
@@ -250,6 +251,10 @@ run: clear
 		"cd ./src; \
 		uvicorn $(PACKAGE_NAME).main:app $(RELOAD) --host 0.0.0.0 --port $(SERVER_PORT_CONTAINER);"
 
+ingest:
+	$(info Running ingestion script to populate database...)
+	export JUPYTER_PORT=$(JUPYTER_PORT); export CONTAINER_USER=$(CONTAINER_USER); export UID=$(UID); export GID=$(GID); docker-compose -p $(APP_NAME) --env-file ./.env run --entrypoint="bash -c \"pip install httpx pandas; cd /home/$(CONTAINER_USER); jupyter nbconvert --to script PopulateDatabase.ipynb; cat PopulateDatabase.py | grep -v get_ipython > run.py; python run.py; rm PopulateDatabase.py run.py\"" jupyter
```

Comment:
Why convert to a script and run instead of just running using nbconvert?

Reply:
I didn't know you could do that!

Reply:
[jupyter nbconvert --to notebook --exec PopulateDatabase.ipynb](https://nbconvert.readthedocs.io/en/latest/execute_api.html#executing-notebooks-from-the-command-line) oughta do it.

Reply:
No, doesn't work:
```
[NbConvertApp] WARNING | Unrecognized alias: 'exec', it will have no effect.
```
but after reading tfm myself I found `--execute` to be working.
Although that gives you no output so I think I prefer the version I stole from [a forum post](https://discourse.jupyter.org/t/how-to-run-a-notebook-using-command-line/3475/12).

Reply:
Aye, you'd get the output back in the notebook. Wanting to have it via stdout suffices as a reason to go this way for me here.